web_accounts_list.json

```diff
@@ -808,6 +808,17 @@
       "category": "misc",
       "valid": true
      },
+     {
+      "name" : "Chyoa",
+      "check_uri" : "https://chyoa.com/user/{account}",
+      "account_existence_code" : "200",
+      "account_existence_string" : "When I'm not reading erotica I like to read",
+      "account_missing_string" : "Sorry, I got distracted...",
+      "account_missing_code" : "404",
+      "known_accounts" : ["joe", "carlos01"],
+      "category" : "XXXPORNXXX",
+       "valid" : true

```

Comment:
Extra space in front of the `"valid" param. Please fix on your next PR. No need to resubmit this.

Reply:
Copy. thanks